flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/utils/StreamAbstractTestBase.java

```diff
@@ -22,10 +22,14 @@
 import org.apache.flink.test.junit5.MiniClusterExtension;
 
 import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** Base class for unit tests that run multiple tests and want to reuse the same Flink cluster. */
 public class StreamAbstractTestBase {
```

Comment:
`StreamAbstractTestBase` is only used by `StreamingTestBase`. Can't we merge the two? :thinking:

Reply:
I made some attempts to put `MiniClusterExtension` in `StreamingTestBase`, but it violated the rules of Arch Unit Test (the scala code could not be well recognized).

https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=55472&view=logs&j=0c940707-2659-5648-cbe6-a1ad63045f0a&t=075c2716-8010-5565-fe08-3c4bb45824a4&l=11485